### PR TITLE
Updated layout for stats.  Added ability to get custom date range.

### PR DESCRIPTION
**Pull request by kablamo**
The main goal here is to add the ability to specify a custom date range when looking at community stats.

I got carried away though and changed the look for the stats.

### CODE REVIEW

**Comment by jbarrett** [comment]
I think we've resolved (or pushed out) any schema conflicts so :+1: if this works, it works :)

(we probably need to talk on that - user github link should probably be a very minimal table with link to github stats tables at some stage)

Assigning to @jagtalon to review styles.

**Comment by jagtalon** [comment]
@kablamo @jbarrett hey thanks will check it soon!

**Comment by kablamo** [comment]
@jbarrett makes sense to me.  Thanks!
@jagtalon i think this page is still ugly.  but i think its good enough.  its not user facing.  but yeah any suggestions for better code are welcome.  i know its a bit sloppy.

File: lib/DDGC/Stats/GitHub.pm [hunk 1]
```diff
@@ -137,22 +137,34 @@ sub forks_report {
...
     while (my $github_issue = $rs->next) {
-        my $first_comment = $github_issue->github_comments->first;
+        my $first_comment = $github_issue->github_comments
+            ->with_github_user_id('!=' => $github_issue->github_user_id)
+            ->order_by('created_at')
+            ->first;
+
         next unless $first_comment;  # FIXME  - should us a duration of zero instead of skipping?
 
+print STDERR sprintf "======================== issue : %s\n", $github_issue->number;
```
Comment (nilnilnil) [inline lib/DDGC/Stats/GitHub.pm]: Isn't `warn` the same thing?

**Comment by jagtalon** [comment]
Looks good to me although I'd like it namespaced just to be sure:

```css
#stats_github {
    .number {
        ...
    }
}
```

And remove the `@import` since it's redundant. Take it away, @jbarrett

**Comment by kablamo** [comment]
ok I did those things @jagtalon asked of me.  Thanks for the feedback!  Now I'm just waiting on Zac to approve this thing.  Once thats done, I'm happy to merge.

**Comment by kablamo** [comment]
@jbarrett if you are are doing a release, I'm happy to see this pr go in.  Feel free to merge.

**Pull request merged**